docs/docs/features/split-keyboards.md

```diff
@@ -0,0 +1,85 @@
+However if you make changes to [config files](../config/index.md) that should apply to all parts, you need to flash to all parts.
+Any changes in ZMK related to split keyboard features might also necessitate doing this.
+:::
+
+## Pairing for Wireless Split Keyboards
+
+Split keyboards with BLE-based split communications (i.e. all officially supported split keyboards) have an internal pairing procedure between the central and each peripheral.
+When the central has an open slot for a peripheral, it will advertise for connections (which will not be visible to non-ZMK devices).
+Then, any peripheral that has not yet bonded to a peripheral will pair to it.
+Similar to how [bluetooth profiles](bluetooth.md) are managed between the keyboard and host devices, the bonding information will be stored with the corresponding hardware addresses of the other keyboard part, on both the central and peripheral.
+
+In practice, this means that your split keyboard parts will automatically pair and work the first time they are all on at the same time.
+However, if this process somehow went wrong or you used controllers in a different split keyboard configuration before, you will need to explicitly clear the stored bond information so that the parts can pair properly.
+For this, please follow [the specified procedure](../troubleshooting/connection-issues.mdx#split-keyboard-halves-unable-to-pair) in the troubleshooting section.
+
+## Behaviors with Locality
+
+ZMK [behaviors](../behaviors/index.mdx) are typically processed and affect the central part of the split keyboard since it handles the keymap state and communication with the host devices.
+However, certain behaviors have different "localities" and need to necessarily affect the peripheral: these include behaviors with global and source localities.
+
+### Global Locality Behaviors
+
+These are behaviors that affect features on all keyboard parts and where their state needs to be explicitly synced between them. Currently these are the following behaviors:
+
+- [RGB underglow behaviors](../behaviors/underglow.md)
+- [Backlight behaviors](../behaviors/backlight.md)
+- [Power management behaviors](../behaviors/power.md)
+- [Soft off behavior](../behaviors/soft-off.md)
+
+### Source Locality Behaviors
+
+These behaviors only affect the keyboard part that they are invoked from, given that they were invoked from a plain behavior binding (i.e. not nested inside another behavior) on a keymap layer. These behaviors include:
+
+- [Reset behaviors](../behaviors/reset.md)
+
+:::note[Peripheral invocation]
+The peripheral side of the keyboard has to be paired and connected to the central side in order to be able to activate these behaviors, even if it is possible to trigger the behavior using only keys on that side. This is because the key bindings are processed on the central side which would then instruct the peripheral side to reset.
```

Comment:
```suggestion
The peripheral side of the keyboard has to be paired and connected to the central side in order to be able to activate these behaviors, even if it is possible to trigger the behavior using only keys on that side. This is because the key bindings are processed on the central side which would then instruct the peripheral side to run the behavior's effect.
```
Copy paste mishap.